packages/npm-package-json-lint-config/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@wordpress/npm-package-json-lint-config",
```

Comment:
Any way we can enforce tabs for indentation? 😉

Reply:
I've requested a new rule to prevent my future self facepalming:

https://github.com/tclindner/npm-package-json-lint/issues/81